Package.swift

```diff
@@ -32,14 +32,19 @@ let package = Package(
     products: [
         .library(
             name: "ArcGIS",
-            targets: ["ArcGIS"]
+            targets: ["ArcGIS", "Runtimecore"]
         )
     ],
     targets: [
         .binaryTarget(
             name: "ArcGIS",
-            url: "https://developers.arcgis.com/ios/downloads/ArcGIS-iOS-v100.12.xcframework.zip",
-            checksum: "15b34f66dec01f1203479a91600a0b81ecb6790ae18b3aa69c8de9f814f506ad"
+            url: "https://developers.arcgis.com/ios/downloads/ArcGIS-iOS-v100.13.xcframework.zip?dl=1",
```

Comment:
Do we really need the `?dl=1` parameter in the URL?

Reply:
Whoops your right we don't need that it was from another link I was using to test